sound/soc/amd/Kconfig

```diff
@@ -61,3 +61,9 @@ config SND_SOC_AMD_ACP5x
 
 	 By enabling this flag build will trigger for ACP PCI driver,
 	 ACP DMA driver, CPU DAI driver.
+
+config SND_AMD_ACP_CONFIG
+	tristate "AMD ACP configuration selection"
+	help
+	 This option figure out machine configuration for ACP
+	 driver modules
```

Comment:
this help text is not very helpful, consider rewording for clarity, e.g. "this option adds an auto-detection to determine which driver to use'

The commit message is equally convoluted and has several spelling/grammar mistakes. Try to stick to 'subject verb complement'